sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/one/PartAUtils.java

```diff
@@ -92,5 +92,8 @@ public static void addPartAFromLog(Log src, CommonSchemaLog dest, String transmi
                 Math.abs(device.getTimeZoneOffset() / 60),
                 Math.abs(device.getTimeZoneOffset() % 60));
         dest.getExt().getLoc().setTz(timezoneOffset);
+
+        /* Add device extension. */
+        dest.getExt().setDevice(new DeviceExtension());
```

Comment:
Please reflect that change in unit tests in some way, like a assert/verify statement.

Reply:
Done. This was supposed to be part #794 :wink: